src/main/java/de/dennisguse/opentracks/util/ExportUtils.java

```diff
@@ -59,6 +59,10 @@ public static DocumentFile getExportDocumentFile(Track.Id trackId, String trackF
         return file;
     }
 
+    public static boolean existsExportDocumentFile(Track.Id trackId, String trackFileFormatExtension, DocumentFile directory) {
```

Comment:
I have no better option, but the name is somehow not telling me what it does.
Something like: `isExportFileExists` (but sucks also) :)